insights/parsers/freeipa_healthcheck_log.py

```diff
@@ -0,0 +1,23 @@
+"""
+FREEIPA HealthCheck log
+================================================
+
+"""
+
+from .. import parser
+from insights.specs import Specs
+from insights import JSONParser
+
+
+@parser(Specs.freeipa_healthcheck_log)
+class FreeIPAHealthCheckLog(JSONParser):
+    """Class to parse /var/log/ipa/healthcheck/healthcheck.log"""
+
+    @property
+    def get_errors(self):
```

Comment:
@fcami normally a property (or attribute) would be named as a noun like `errors` instead of a verb.

Reply:
Will fix, thanks.